UndertaleModTool/Editors/UndertaleSpriteEditor.xaml.cs

```diff
@@ -1,224 +1,223 @@
+                    if (mainWindow.ShowQuestion("Include padding?") == MessageBoxResult.Yes)
+                        includePadding = true;
+
+                    if (sprite.Textures.Count > 1)
+                    {
+                        string dir = Path.GetDirectoryName(dlg.FileName);
+                        string name = Path.GetFileNameWithoutExtension(dlg.FileName);
+                        string path = Path.Combine(dir, name);
+                        string ext = Path.GetExtension(dlg.FileName);
+
+                        Directory.CreateDirectory(path);
+                        foreach (var tex in sprite.Textures.Select((tex, id) => new { id, tex }))
+                        {
+                            try
+                            {
+                                worker.ExportAsPNG(tex.tex.Texture, Path.Combine(path, sprite.Name.Content + "_" + tex.id + ext), null, includePadding);
+                            }
+                            catch (Exception ex)
+                            {
+                                mainWindow.ShowError("Failed to export file: " + ex.Message, "Failed to export file");
+                            }
+                        }
+                    }
+                    else if (sprite.Textures.Count == 1)
+                    {
+                        try
+                        {
+                            worker.ExportAsPNG(sprite.Textures[0].Texture, dlg.FileName, null, includePadding);
+                        }
+                        catch (Exception ex)
+                        {
+                            mainWindow.ShowError("Failed to export file: " + ex.Message, "Failed to export file");
+                        }
+                    }
+                    else
+                    {
+                        mainWindow.ShowError("No frames to export", "Failed to export sprite");
+                    }
+                }
+                catch (Exception ex)
+                {
+                    mainWindow.ShowError("Failed to export: " + ex.Message, "Failed to export sprite");
+                }
+            }
+
+            worker.Cleanup();
+        }
+
+        private void MaskList_AddingNewItem(object sender, AddingNewItemEventArgs e)
+        {
+            e.NewItem = (this.DataContext as UndertaleSprite).NewMaskEntry();
+        }
+
+        private void MaskImport_Click(object sender, RoutedEventArgs e)
+        {
+            UndertaleSprite sprite = this.DataContext as UndertaleSprite;
+            UndertaleSprite.MaskEntry target = (sender as Button).DataContext as UndertaleSprite.MaskEntry;
+
+            OpenFileDialog dlg = new OpenFileDialog();
+            dlg.DefaultExt = ".png";
+            dlg.Filter = "PNG files (.png)|*.png|All files|*";
+
+            if (dlg.ShowDialog() == true)
+            {
+                try
+                {
+                    System.Drawing.Image img = System.Drawing.Image.FromFile(dlg.FileName);

```

Comment:
the system.drawing using reference was removed, but the usages in the file were still left. intended?

Reply:
Not, it's not, I should fix it.